.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Integrate
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: python_setup
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8 
+    - name: before_install
+      run: |
+        sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+        sudo apt-get -q update
+        sudo apt-get install build-essential git libboost-all-dev cmake libgmp3-dev libssl-dev libsodium-dev libprocps-dev pkg-config gcc-5 g++-5
+        sudo apt-get install clang-tidy
+        mkdir build
+    - name: wget_clang_tidy_script
+      uses: wei/wget@v1
+      with:
+        args: https://raw.githubusercontent.com/llvm/llvm-project/master/clang-tools-extra/clang-tidy/tool/run-clang-tidy.py -P ./build
+    - name: before_script
+      run: |
+        git submodule init && git submodule update
+        cd build
+        PATH=$PATH:${PWD} 
+        sudo chmod +x run-clang-tidy.py
```

Comment:
Does this need sudo?

Reply:
This is a workaround from installing clang-tidy using wget. The newest iteration does not include this.